packages/rps/src/redux/store.ts

```diff
@@ -26,9 +27,7 @@ function* rootSaga() {
   const channelProviderEnabled = yield stateChannelWalletSaga();
 
   if (channelProviderEnabled) {
-    const client = new RPSChannelClient();
-
-    yield call([client, client.enable]);
+    const client = new RPSChannelClient(new ChannelClient(window.channelProvider));
```

Comment:
These lines encapsulate fairly well the new approach (of this PR) to instantiating the double wrapper around `window.channelProvider`. There were comments in the existing code intimating that this was the desired approach in any case, and it also makes testing far easier.